sites/authentication/contextual-signin-en.html

```diff
@@ -0,0 +1,31 @@
+---
+{
+	"altLangPage": "contextual-signin-fr.html",
+	"breadcrumb": {
+		"title": "Canada.ca", "link": "https://www.canada.ca/en.html"
+	},
+	"dateModified": "2022-05-02",
+	"description": "Example of the contextual sign-in button with <code>label</code>as custom label only.",
+	"language": "en",
+	"title": "Contextual Sign-in button",
+	"auth": {
+		"type": "contextual",
+		"link": "https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/account-individuals.html",
+		"label": "CRA Account"
+	}
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<h2>Front matter example</h2>
```

Comment:
Be specific this is only applicable to our GCWeb jekyll CMS
```suggestion
<h2>GCWeb Jekyll Front matter example</h2>
```